source/material_model/reactive_fluid_transport.cc

```diff
@@ -71,6 +67,10 @@ namespace aspect
       // Td polynomials are defined in equations 15, B3, B11, and B19.
       for (unsigned int i = 0; i<devolatilization_enthalpy_changes.size(); ++i)
         {
+          // Pressure, which must be in GPa for the parametrization, or GPa^-1. The polynomials for each lithology
+          // breaks down above certain pressures, make sure that we cap the pressure just before this break down.
+          const double pressure = std::min(std::max(1e-12, in.pressure[q]/1.e9), pressure_cutoffs[i]);
```

Comment:
What is 1e-12? Can you introduce a variable for it and give it a name reflecting this?

Reply:
It is just a small value that is close to 0 that prevents a division by 0 in the next line

Reply:
@tjhei - would it be more self-explanatory to use `numeric_limits::min()` instead of 1e-12, and/or add a comment?